src/leafletwidget/widget.ts

```diff
@@ -131,7 +145,9 @@ class MapWidget extends Widget {
     this.update();
   }
 
-  private _geojsonString = '';
+  private _width: number = -1;
```

Comment:
Thanks. I originally had null, but then didn't remove the type when I switched to -1 as the unknown option.